examples/todo-list/package.json

```diff
@@ -21,11 +21,11 @@
     "prettier:fix": "npm run prettier:cli -- --write",
     "tslint": "lb-tslint",
     "tslint:fix": "npm run tslint -- --fix",
-    "pretest": "npm run build:current",
+    "pretest": "npm run build",
```

Comment:
Is this rename in line with the rest of our packages/examples?

Reply:
yup